src/parse.c

```diff
@@ -952,19 +958,19 @@ StorageClass Parser::appendStorageClass(StorageClass storageClass, StorageClass
     {
         StorageClass u = storageClass & (STCconst | STCimmutable | STCmanifest);
         if (u & (u - 1))
-            error("conflicting attribute '%s'", Token::toChars(token.value));
+            error("conflicting attribute '%s'", Token::toChars(token.value), deprec);
     }
     if (stc & (STCgshared | STCshared | STCtls))
     {
         StorageClass u = storageClass & (STCgshared | STCshared | STCtls);
         if (u & (u - 1))
-            error("conflicting attribute '%s'", Token::toChars(token.value));
+            error("conflicting attribute '%s'", Token::toChars(token.value), deprec);
     }
     if (stc & (STCsafe | STCsystem | STCtrusted))
     {
         StorageClass u = storageClass & (STCsafe | STCsystem | STCtrusted);
         if (u & (u - 1))
-            error("conflicting attribute '@%s'", token.toChars());
+            error("conflicting attribute '@%s'", token.toChars(), deprec);
```

Comment:
Yes, this is intentional (and I have mentioned that in earlier comment). Contrary to redundant attributes, conflicting ones don't have safe and meaningful interpretation so it makes more sense to straight out prohibit those as that is a clear indicator of an error in program (as opposed to just sloppy coding style).

You would prefer to deprecate both?